ubiquibot-config-default.yml

```
@@ -0,0 +1,41 @@
+---
+chain-id: 1
+base-multiplier: 1000
+issue-creator-multiplier: 2000
+time-labels:
+  - name: "Time: <1 Hour"
+    weight: 0.125
+    value: 3600
+  - name: "Time: <1 Day"
+    weight: 1
+    value: 86400
+  - name: "Time: <1 Week"
+    weight: 2
+    value: 604800
+  - name: "Time: <2 Weeks"
+    weight: 3
+    value: 1209600
+  - name: "Time: <1 Month"
+    weight: 4
+    value: 2592000
+priority-labels:
+  - name: "Priority: 0 (Normal)"
+    weight: 1
+  - name: "Priority: 1 (Medium)"
+    weight: 2
+  - name: "Priority: 2 (High)"
+    weight: 3
+  - name: "Priority: 3 (Urgent)"
+    weight: 4
+  - name: "Priority: 4 (Emergency)"
+    weight: 5
+auto-pay-mode: true
+analytics-mode: true
+incentive-mode: false
+max-concurrent-bounties: 2
+comment-element-pricing:
+  text: 0.1
+  link: 0.5
+  list: 0.5
+  code: 5
+  image: 5
```

```suggestion
  text: 0.1
  link: 0.5
  list: 0.5
  code: 5
  image: 5
  italics: 0
```

The only time I've seen italics used on a normal basis are in the auto generated citation. For example:

> _Originally posted by @pavlovcik in https://github.com/ubiquity/ubiquibot/issues/410#issuecomment-1621438825_

Example: https://github.com/ubiquity/ubiquibot/issues/468#issue-1789931394